src/nyc_trees/nyc_trees/settings/production.py

```diff
@@ -83,7 +83,7 @@ def get_env_setting(setting):
 AWS_STORAGE_BUCKET_NAME = 'django-storages-{}'.format(vpc_id)
 
 AWS_AUTO_CREATE_BUCKET = True
-DEFAULT_FILE_STORAGE = 'storages.backends.s3boto.S3BotoStorage'
+DEFAULT_FILE_STORAGE = 'libs.custom_storages.PublicS3BotoStorage'
```

Comment:
In testing, I followed @jwalgran's existing convention of placing this custom backend in `apps/core/storage/backends/s3boto.py`.